www/src/components/shared/pullquote.js

```diff
@@ -155,13 +163,14 @@ const Star = styled(`span`)`
 const variants = [`A`, `B`, `C`]
 let instancesCounter = -1
 
-const Pullquote = ({ children }) => {
+const Pullquote = ({ citation, children }) => {
   instancesCounter += 1
   const className = `variant${variants[instancesCounter % variants.length]}`
 
   return (
     <PullquoteRoot className={className}>
       {children}
+      {citation && <Citation>- {citation}</Citation>}
```

Comment:
nit: but should this be an emdash or endash maybe?

```suggestion
       {citation && <Citation>&mdash;{citation}</Citation>}
```